src/main/webapp/components/BuildSnapshot.js

```diff
@@ -26,6 +26,7 @@ import toggleIcon from './resources/play.svg';
  * @param {string} props.buildData[].commitHash - Commit hash.
  * @param {string} props.buildData[].repository - URL to the GitHub repository.
  * @param {string} props.buildData[].status - The status of the build.
+ * Either 'passed', 'failed' or 'lost'.
```

Comment:
[line wrapping in JSDoc adds +4 spaces](https://google.github.io/styleguide/jsguide.html#jsdoc-line-wrapping)